vana/client.py

```diff
@@ -149,3 +149,19 @@ def claim(self):
         """
         claim_fn = self.tee_pool_contract.functions.claim()
         return self.chain_manager.send_transaction(claim_fn, self.wallet.hotkey)
+
+    def get_jobs_count(self) -> int:
+        """
+        Get the total number of jobs from the TEE Pool contract
+
+        Returns:
+            int: Total number of jobs in the queue
+        """
+        try:
+            # Call the TEE Pool contract's jobCount method
+            job_count = self.tee_pool_contract.functions.jobsCount().call()
```

Comment:
Can we use chain_manager.read_contract_fn, it has built in error decoding logic. Ex: 

```
job_count_fn = self.tee_pool_contract.functions.jobsCount()
job_count = self.chain_manager.read_contract_fn(job_count_fn)
```